modules/cmp/impl/clusters/credential.go

```diff
@@ -24,24 +24,24 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
-	credentialpb "github.com/erda-project/erda-proto-go/core/services/authentication/credentials/accesskey/pb"
+	"github.com/erda-project/erda-proto-go/core/token/pb"
 	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/modules/cmp/conf"
 	"github.com/erda-project/erda/modules/cmp/dbclient"
 	"github.com/erda-project/erda/pkg/k8sclient"
+	"github.com/erda-project/erda/pkg/oauth2/tokenstore/mysqltokenstore"
 )
 
 // GetAccessKey get access key with cluster name.
-func (c *Clusters) GetAccessKey(clusterName string) (*credentialpb.QueryAccessKeysResponse, error) {
-	return c.credential.QueryAccessKeys(context.Background(), &credentialpb.QueryAccessKeysRequest{
-		Status:      credentialpb.StatusEnum_ACTIVATE,
-		SubjectType: credentialpb.SubjectTypeEnum_CLUSTER,
-		Subject:     clusterName,
```

Comment:
what difference between queryAccessKeys and QueryTokens, there Request look so different.

Reply:
Cluster manage uses clusterName as ID. Subject and Subject type are deprecated. We use scope and scope id instead.

Reply:
okay